hardcodes.R

```diff
@@ -20,9 +20,9 @@ allowed_destinations <-
   c(116, 101, 118, 215, 206, 207, 225, 204, 205, 302, 329, 314, 332, 312, 313,
     327, 422, 423, 426, 410, 435, 421, 411, 30, 17, 24, 8, 9, 99)
 
-perm_destinations <- c(422,423,426,410,435,421,411)
+perm_destinations <- c(400:499)
```

Comment:
Curious on your thoughts, Genelle, on whether separate vectors are needed for _destinations and _livingsituation or if having them combined into vectors like perm_situation, temp_situation, lh_situation etc. would be useful and simplify things further.

This approach uses the same terminology as the Data Standards and would rely on whatever data field is referenced in the code (3.917, 4.12, or 3.12) to determine which of the values in the list to pull. 

For example, if some code said something like "IF Enrollment.LivingSituation IN (perm_situation) THEN blah blah blah," we would expect it not to include values 422, 423, or 426 out of the range 400:499 since those three values should only be options in an HMIS for destinations.

That said, one of the things I like about having the vectors separate is just the clarity around the vector names, even if the values are the same across the vectors (i.e., perm_destinations = 400:499 and perm_livingsituation = perm_destinations).

Reply:
Hi @vlopez0603, the assumption behind including the entire set of permanent living situations is that the data is coming out of HMIS in the way it's collected, so a value like 422 wouldn't be represented in the Destination data element. Looking for it wouldn't cause any harm if the assumption can be made that HMIS users are not allowed to select the 422 value in the Destination drop list in their software. If we're worried about this, we could continue to list them out each individually. I have not personally seen this problem in the SSVF data I work with or any of the sample data sets we've gotten. I guess if that was happening, what would we do with that value? Filter it out or count it as missing or would we consider it permanent and notify the community? 
If we are good with assuming that there won't be 422's (as an example) in Destination data and that if there were that we'd count them as permanent, then it is efficient to use this method since as future data standards come out, we won't have additional values to add to the lists, they would already be categorized by their value as to whether they're permanent, temporary, etc.

Reply:
Also I just noticed that in the FSA, we do check for valid Prior Living Situation and Destination values, so if those values are coming in the wrong places, they are flagged.

Reply:
Hi again, :) I noticed that we don't use "temp_destinations" anywhere so I'll remove that and we can just use temp_livingsituation. 
We do use both perm_destinations and perm_livingsituation, but I think to keep things consistent, we should switch to using perm_livingsituation even though perm_destinations is such an icon, haha. It just seems cleaner and less confusing for other devs to do it that way.

Reply:
Agree with making the switch to using just the _livingsituation vectors for consistency and efficiency.